backend/tests/unit/hal/test_core_tei.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'd like to see a real test (with VCR) where we push a record to HAL, at least for the happy flow. Currently we have no idea if this whole thing is working at all.